docs/how-to-guides/develop-a-tool/use-file-path-as-tool-input.md

```diff
@@ -0,0 +1,98 @@
+# Use FilePath as tool input
+
+In this document, we will guide you through the detailed process of how to use FilePath as tool input with 'hello-world' tool example.
+
+## Prerequisites
+
+Since FilePath is still in test currently, you need to install private version SDK and Extension.
+
+1. Download private version promptflow SDK
+
+    ```bash
+    pip install "promptflow==0.0.106113823" --extra-index-url https://azuremlsdktestpypi.azureedge.net/test-promptflow/
+    ```
+
+2. Download this [VSCode Extension](https://aka.ms/pfvsctest), install from VSIX:
+
+    ![install from vsix](../../media/how-to-guides/develop-a-tool/install_from_vsix.png)
+
+## Use FilePath as tool input in 'hello-world' tool
+
+For details of how to create and use custom tool, please refer to [Custom tool package creation and usage](../how-to-create-and-use-your-own-tool-package.md).
+
+1. Create a tool project hello-world-proj with `--case file_input` to generate demo case of file_path as tool input.
```

Comment:
1-4 are all steps of tool package creation, which can be replaced simply by a reference link to create tool package doc.
Would suggest add an example tool to https://github.com/microsoft/promptflow/tree/main/examples/tools/tool-package-quickstart/my_tool_package/tools, and start from the tool definition and show what tool author should do.

Reply:
Add a tool to it.